sympy/printing/tests/test_latex.py

```diff
@@ -149,6 +157,7 @@ def test_latex_symbols():
     assert latex(Symbol('omega') ** Symbol('beta')) == r"\omega^{\beta}"
 
 
+
```

Comment:
In trivial cases like this, it probably makes sense to squash it. If it is a bigger PR, you should always keep adding patches, otherwise it is very hard for reviewers to see what was updated.